release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -814,6 +814,14 @@ If you disable the `DeploymentConfig` cluster capability, the `deployer` service
 
 For more information, see xref:../installing/cluster-capabilities.adoc#deployment-config-capability_cluster-capabilities[DeploymentConfig capability].
 
+[discrete]
+[id="ocp-4-15-must-gather-storage-default"]
+=== Must-gather storage limit default
+
+ A default limit of 30% of the storage capacity of the node for the container has been added for data collected by the `oc adm must-gather` command. If necessary, you can use the `--volume-percentage` flag to adjust the default storage limit.
+
+For more information, see xref:../support/gathering-cluster-data.adoc#must-gather-storage-limit_gathering-cluster-data[Changing the must-gather storage limit].
```

Comment:
- This xref doesn't seem to be opening to the correct location for me in the rendered preview.

Reply:
The section it links to only exists in the 4.15 branch so I _think_ that means it won't show up properly in the preview.

Reply:
Might be something in my Firefox browser settings. It appears to be working OK in Chrome. Setting this to resolved.